### ISSUE TITLE
Focus outline color is not accessible in light mode

### ISSUE THREAD

**Issue by seedy** [traefik/faency#191]
### Description

- Focus outline color: `#4791ff`
- ContentBg: 
  - Light mode: `#f2f2f3`
  - Dark mode: `#071727`

- [WCAG contrast in light mode](https://webaim.org/resources/contrastchecker/?fcolor=4791FF&bcolor=F2F2F3): 
  - Graphical Objects and User Interface Components: WCAG AA:  Fail : :negative_squared_cross_mark: 

- [WCAG contrast in dark mode](https://webaim.org/resources/contrastchecker/?fcolor=4791FF&bcolor=071727):
  - Graphical Objects and User Interface Components: WCAG AA: pass :heavy_check_mark:

**Comment by seedy** [comment]
@matthieuh Could you please have a look ?

:information_source: `$focusOutline` is used for `Button` (I don't see this color in the mockups?) and `Badge`
